examples/openapi-tools/quickstart-se/README.md

```diff
@@ -0,0 +1,352 @@
+    }
+}
+```
+
+3) Add class `Main` that will be the main class for our client :
+```java
+public final class Main {
+
+    /**
+     * Cannot be instantiated.
+     */
+    private Main() {
+    }
+
+    /**
+     * Application main entry point.
+     * @param args command line arguments.
+     */
+    public static void main(final String[] args) {
+        startServer();
+    }
+
+    /**
+     * Start the server.
+     * @return the created {@link WebServer} instance
+     */
+    static Single<WebServer> startServer() {
+
+        // load logging configuration
+        LogConfig.configureRuntime();
+
+        // By default this will pick up application.yaml from the classpath
+        Config config = Config.create();
+
+        WebServer server = WebServer.builder(createRouting(config))
+                                    .config(config.get("server"))
+                                    .addMediaSupport(JacksonSupport.create())
+                                    .build();
+
+        Single<WebServer> webserver = server.start();
+
+        // Try to start the server. If successful, print some info and arrange to
+        // print a message at shutdown. If unsuccessful, print the exception.
+        webserver.thenAccept(ws -> {
+                     System.out.println("WEB server is up! http://localhost:8081");
+                     ws.whenShutdown().thenRun(() -> System.out.println("WEB server is DOWN. Good bye!"));
+                 })
+                 .exceptionallyAccept(t -> {
+                     System.err.println("Startup failed: " + t.getMessage());
+                     t.printStackTrace(System.err);
+                 });
+
+        return webserver;
+    }
+
+    /**
+     * Creates new {@link Routing}.
+     *
+     * @return routing configured with JSON support, a health check, and a service
+     * @param config configuration of this server
+     */
+    private static Routing createRouting(Config config) {
+
+        return Routing.builder()
+                      .register("/", new MessageService())
+                      .build();
+    }
+}
+```
+
+4) Create `resource` folder and put a `application.yaml` file inside.
+```yaml
+server:
+  port: 8081
+  host: localhost
+```
+
+To run the application :
+
+With JDK17+
+```bash
+mvn package
+java -jar target/openapi-se-client.jar
+```
+
+To check that client works as expected and process all the request using our server run the following `curl` commands :
+
+```
+curl -X PUT -H "Content-Type: application/json" -d '{"greeting" : "Hola", "message":"Lisa"}' http://localhost:8081/greet/greeting
+
+curl -X GET http://localhost:8081/greet
+{"message":"Lisa","greeting":"Hola"}
+
+curl -X GET http://localhost:8081/greet/Joe
+{"message":"Joe","greeting":"Hola"}
+```
+
+## Update applications
+
+To keep the server and the client up to date according to the OpenApi document, we can use the maven plugin.
```

Comment:
Please add instructions showing how users can run the profile to run the plug-in to generate the code. Maybe something like the next line:

To run the generator during your build, invoke the profile: `mvn clean package -P openapi`